python-checks/src/test/resources/checks/changeMethodContract/changeMethodContractIntermediate.py

```diff
@@ -0,0 +1,8 @@
+from changeMethodContractParent import ParentClass
+from typing import Any
+
+class IntermediateClass(ParentClass):
+  ...
+
+class IntermediateWithAnyClass(ParentClass[Any]):
```

Comment:
does it make a difference in case we have an extra file here compared to just having these classes declared in the `changeMethodContractParent.py`?